mrmustard/utils/settings.py

```diff
@@ -122,6 +124,14 @@ def __init__(self):
         self.ATOL: float = 1e-8
         "The absolute tolerance when comparing two values or arrays. Default is 1e-8."
 
+        self._frozen = True
```

Comment:
When would we have frozen be false? And does that mean then the user could populate with their own settings?

Reply:
it's defined as `False` at the class level, then while `__init__` happens and sets all the default values, there's no issue. but at the very end of `__init__` (here), we set it to True forevermore. I've seen this before, I think it's the semi-standard way of locking a class